google/cloud/bigtable/table.py

```diff
@@ -1083,7 +1079,12 @@ def __call__(self, retry=DEFAULT_RETRY):
 
     @staticmethod
     def _is_retryable(status):
-        return status is None or status.code in _RetryableMutateRowsWorker.RETRY_CODES
+        RETRY_CODES = tuple(
+            retryable.grpc_status_code.value[0]
+            for retryable in RETRYABLE_MUTATION_ERRORS
+        )
```

Comment:
Seems like a performance hit here to recompute every time.

Reply:
Yeah. This is a bit irritating. So I tried adding an if clause to the list comp, but the error was still there. If I structured it as a regular for loop to a list, then cast to tuple, it likely will pass, but that is a bit.. gross?